nightly-metrics-update.sh

```diff
@@ -35,6 +35,9 @@ cd ../osg-cpu-hours
 cd ../osg-project-waittime
 ./calculate-waittime.py ../osg-waittime.csv $START_DATE $END_DATE
 
+cd ../software
```

Comment:
It might be nice to have a more specific directory name for this than "software", which sounds a bit generic.

What is this metric about?  Something about jira issues, i suppose.

Reply:
This metric is about how often software issues change due dates. I agree about a better directory name, and generally better organization of these metric scripts. But I don't think this is the time for it. Right now the "software" folder contains all scripts used by the software team, since "flightworthy" is now verboten.

Reply:
Note that the HTCSS dev team is the "software" arm of PATh so this makes sense, even though it's generic sounding.